campaignion_email_to_target/campaignion_email_to_target.module

```diff
@@ -153,35 +155,70 @@ function campaignion_email_to_target_token_info() {
       drupal_set_message(t('We are sorry! The service is temporary unavailable. The administrators have been informed. Please try again in a few minutes …'), 'error');
     }
   }
+
+  $info['tokens']['email-to-target']['messages'] = [
```

Comment:
If we put the token in the submission group it would be automatically available on thank you pages for example. I’d reserve the `email-to-target` tokens for stuff that can be replaced inside of and per message.